packages/vm/src/imports.rs

```diff
@@ -232,6 +256,27 @@ fn do_humanize_address<A: Api, S: Storage, Q: Querier>(
     }
 }
 
+fn do_secp256k1_verify<A: Api, S: Storage, Q: Querier>(
+    env: &Environment<A, S, Q>,
+    hash_ptr: u32,
+    signature_ptr: u32,
+    pubkey_ptr: u32,
+) -> VmResult<u32> {
+    let hash = read_region(&env.memory(), hash_ptr, LENGTH_SHA256_HASH)?;
+
+    let signature = read_region(&env.memory(), signature_ptr, MAX_LENGTH_SIGNATURE)?;
+
+    let pubkey = read_region(&env.memory(), pubkey_ptr, MAX_LENGTH_PUBKEY)?;
+
+    let result = secp256k1_verify(&hash, &signature, &pubkey);
+    let gas_info = GasInfo::with_cost(GAS_COST_VERIFY_SECP256K1_SIGNATURE);
+    process_gas_info::<A, S, Q>(env, gas_info)?;
+    match result {
+        Ok(b) => Ok(b.into()),
+        Err(err) => Err(VmError::crypto_err(err)),
+    }
```

Comment:
And now this show work :)

```suggestion
    Ok(result?.into())
```

Reply:
Thanks. :-)